### DISCUSSION TITLE
Command to turn off pc!

### DISCUSSION THREAD

**Discussion by chiconanni** [Reetus/ClassicAssist#292] [Ideas]
With the addition of the logout() and quit() commands, i think an another good one will be the TurnOffPc() command! That will be awesome!

**Comment by d0tsd0tsd0ts** [comment]
u can do it, executing some .bat by python using modules

**Comment by chiconanni** [comment]
i would do it, if i know how to! i know to create a .bat using windows internal tools to turn-off in xxxxx seconds, but lets say if the skill that you are training reach the cap, then shutdown pc, to use scripts to trains, with the other two ones will be a good trio!

**Comment by Lloydy86** [comment]
an easy way would be to 
import os
os.system("shutdown /s /t 1")
